src/commands/project/add.js

```diff
@@ -1,16 +1,10 @@
 // @flow
-import * as options from '../../utils/options';
-import { BoltError } from '../../utils/errors';
+import addDependenciesToPackage from '../../utils/addDependenciesToPackages';
+import Project from '../../Project';
+import { type AddOptions } from '../add';
 
-export type ProjectAddOptions = {};
-
-export function toProjectAddOptions(
-  args: options.Args,
-  flags: options.Flags
-): ProjectAddOptions {
-  return {};
-}
-
-export async function projectAdd(opts: ProjectAddOptions) {
-  throw new BoltError('Unimplemented command "project add"');
+export async function projectAdd(opts: AddOptions) {
+  let cwd = opts.cwd || process.cwd();
+  let project = await Project.init(cwd);
+  await addDependenciesToPackage(project, project.pkg, opts.deps, opts.type);
```

Comment:
I believe this is what you're saying above, but to reiterate, this should just be `yarn.add` in the project.
Then the test just needs to check that yarn.add got called with its `cwd` set to projects directory